sycl/doc/extensions/USM/USM.adoc

```diff
@@ -299,26 +331,36 @@ Parameters::
   * `size_t count` - number of bytes to copy
 Return value:: Returns an event representing the copy operation.
 
-===== memset
+===== memfill
 [source,cpp]
 ----
 class handler {
  ...
  public:
   ...
-  void memset(void* ptr, int value, size_t count);
```

Comment:
Is there a reason to replace `memset` with `memfill` instead of providing both?  I've never heard of the latter, and if I wanted to zero a buffer my first thought would be to look for `memset` support.

Reply:
The CL extension made this change.  Generally memfill is more generic (and usable) than memset.

Reply:
SYCL buffers also already support fill.

Reply:
The `copy`/`fill` functions provided by buffer are consistent with `<algorithm>`, and `memcpy`/`memset` are consistent with `<string>`.  `memfill` is neither.

I'm not against introducing a new `memfill` function with nicer semantics, but some developers (like me) might be surprised to see `memcpy` without `memset`.

Reply:
Wouldn't it make sense to name it `fill`, for consistency with the existing accessor `fill`? It has the same semantics.

Also is the non-templated version required? The existing `fill` expects `T` to be a scalar or a SYCL vector type.